2024-03-02 09:54:48,214 - __main__ - INFO - setData function - AQI is not an integer. Exception ValueError has occured for station=> nise gwal
2024-03-02 09:54:48,910 - __main__ - INFO - station=> Sector-51, Gurugram, result => ['103', 'Sector-51, Gurugram, India', Timestamp('2024-03-02 15:00:00')]
2024-03-02 09:54:48,910 - __main__ - INFO - The hourly data has been written to data/cron_job_data/sector_51_cron_output.csv with Timestamp: 2024-03-02 15:00:00
2024-03-02 09:54:49,591 - __main__ - INFO - station=> Sector-51, Gurugram, result => ['103', 'Sector-51, Gurugram, India', Timestamp('2024-03-02 15:00:00')]
2024-03-02 09:54:49,591 - __main__ - INFO - The hourly data has been written to static/data/sector_51_cron_output.csv with Timestamp: 2024-03-02 15:00:00
2024-03-02 09:54:50,292 - __main__ - INFO - station=> Teri Gram, result => ['173', 'Teri Gram, Gurugram, India', Timestamp('2024-03-02 15:00:00')]
2024-03-02 09:54:50,292 - __main__ - INFO - The hourly data has been written to data/cron_job_data/teri_gram_cron_output.csv with Timestamp: 2024-03-02 15:00:00
2024-03-02 09:54:50,988 - __main__ - INFO - station=> Vikas Sadan, result => ['114', 'Vikas Sadan Gurgaon, Gurgaon, India', Timestamp('2024-03-02 14:00:00')]
2024-03-02 09:54:50,988 - __main__ - INFO - The hourly data has been written to data/cron_job_data/vikas_sadan_cron_output.csv with Timestamp: 2024-03-02 14:00:00
2024-03-02 10:16:04,202 - __main__ - INFO - setData function - AQI is not an integer. Exception ValueError has occured for station=> nise gwal
2024-03-02 11:15:01,387 - __main__ - INFO - setData function - AQI is not an integer. Exception ValueError has occured for station=> nise gwal
2024-03-02 11:15:01,863 - __main__ - INFO - station=> Sector-51, Gurugram, result => ['93', 'Sector-51, Gurugram, India', Timestamp('2024-03-02 16:00:00')]
2024-03-02 11:15:01,863 - __main__ - INFO - The hourly data has been written to data/cron_job_data/sector_51_cron_output.csv with Timestamp: 2024-03-02 16:00:00
2024-03-02 11:15:02,337 - __main__ - INFO - station=> Sector-51, Gurugram, result => ['93', 'Sector-51, Gurugram, India', Timestamp('2024-03-02 16:00:00')]
2024-03-02 11:15:02,337 - __main__ - INFO - The hourly data has been written to static/data/sector_51_cron_output.csv with Timestamp: 2024-03-02 16:00:00
2024-03-02 11:15:02,816 - __main__ - INFO - station=> Teri Gram, result => ['175', 'Teri Gram, Gurugram, India', Timestamp('2024-03-02 16:00:00')]
2024-03-02 11:15:02,816 - __main__ - INFO - The hourly data has been written to data/cron_job_data/teri_gram_cron_output.csv with Timestamp: 2024-03-02 16:00:00
2024-03-02 12:28:24,118 - __main__ - INFO - setData function - AQI is not an integer. Exception ValueError has occured for station=> nise gwal
2024-03-02 12:28:24,807 - __main__ - INFO - station=> Sector-51, Gurugram, result => ['93', 'Sector-51, Gurugram, India', Timestamp('2024-03-02 17:00:00')]
2024-03-02 12:28:24,807 - __main__ - INFO - The hourly data has been written to data/cron_job_data/sector_51_cron_output.csv with Timestamp: 2024-03-02 17:00:00
2024-03-02 12:28:25,494 - __main__ - INFO - station=> Sector-51, Gurugram, result => ['93', 'Sector-51, Gurugram, India', Timestamp('2024-03-02 17:00:00')]
2024-03-02 12:28:25,495 - __main__ - INFO - The hourly data has been written to static/data/sector_51_cron_output.csv with Timestamp: 2024-03-02 17:00:00
2024-03-02 12:28:26,194 - __main__ - INFO - station=> Teri Gram, result => ['177', 'Teri Gram, Gurugram, India', Timestamp('2024-03-02 17:00:00')]
2024-03-02 12:28:26,194 - __main__ - INFO - The hourly data has been written to data/cron_job_data/teri_gram_cron_output.csv with Timestamp: 2024-03-02 17:00:00
2024-03-02 12:28:26,892 - __main__ - INFO - station=> Vikas Sadan, result => ['88', 'Vikas Sadan Gurgaon, Gurgaon, India', Timestamp('2024-03-02 17:00:00')]
2024-03-02 12:28:26,892 - __main__ - INFO - The hourly data has been written to data/cron_job_data/vikas_sadan_cron_output.csv with Timestamp: 2024-03-02 17:00:00
2024-03-02 13:16:32,393 - __main__ - INFO - setData function - AQI is not an integer. Exception ValueError has occured for station=> nise gwal
2024-03-02 13:16:33,092 - __main__ - INFO - station=> Sector-51, Gurugram, result => ['106', 'Sector-51, Gurugram, India', Timestamp('2024-03-02 18:00:00')]
2024-03-02 13:16:33,092 - __main__ - INFO - The hourly data has been written to data/cron_job_data/sector_51_cron_output.csv with Timestamp: 2024-03-02 18:00:00
2024-03-02 13:16:33,789 - __main__ - INFO - station=> Sector-51, Gurugram, result => ['106', 'Sector-51, Gurugram, India', Timestamp('2024-03-02 18:00:00')]
2024-03-02 13:16:33,789 - __main__ - INFO - The hourly data has been written to static/data/sector_51_cron_output.csv with Timestamp: 2024-03-02 18:00:00
2024-03-02 13:16:34,493 - __main__ - INFO - station=> Teri Gram, result => ['182', 'Teri Gram, Gurugram, India', Timestamp('2024-03-02 18:00:00')]
2024-03-02 13:16:34,493 - __main__ - INFO - The hourly data has been written to data/cron_job_data/teri_gram_cron_output.csv with Timestamp: 2024-03-02 18:00:00
2024-03-02 13:16:35,208 - __main__ - INFO - station=> Vikas Sadan, result => ['143', 'Vikas Sadan Gurgaon, Gurgaon, India', Timestamp('2024-03-02 18:00:00')]
2024-03-02 13:16:35,208 - __main__ - INFO - The hourly data has been written to data/cron_job_data/vikas_sadan_cron_output.csv with Timestamp: 2024-03-02 18:00:00
2024-03-02 14:14:37,159 - __main__ - INFO - setData function - AQI is not an integer. Exception ValueError has occured for station=> nise gwal
2024-03-02 14:14:37,989 - __main__ - INFO - station=> Sector-51, Gurugram, result => ['99', 'Sector-51, Gurugram, India', Timestamp('2024-03-02 19:00:00')]
2024-03-02 14:14:37,989 - __main__ - INFO - The hourly data has been written to data/cron_job_data/sector_51_cron_output.csv with Timestamp: 2024-03-02 19:00:00
2024-03-02 14:14:38,622 - __main__ - INFO - station=> Sector-51, Gurugram, result => ['99', 'Sector-51, Gurugram, India', Timestamp('2024-03-02 19:00:00')]
2024-03-02 14:14:38,622 - __main__ - INFO - The hourly data has been written to static/data/sector_51_cron_output.csv with Timestamp: 2024-03-02 19:00:00
2024-03-02 14:14:39,306 - __main__ - INFO - station=> Teri Gram, result => ['171', 'Teri Gram, Gurugram, India', Timestamp('2024-03-02 19:00:00')]
2024-03-02 14:14:39,307 - __main__ - INFO - The hourly data has been written to data/cron_job_data/teri_gram_cron_output.csv with Timestamp: 2024-03-02 19:00:00
2024-03-02 14:14:39,943 - __main__ - INFO - station=> Vikas Sadan, result => ['160', 'Vikas Sadan Gurgaon, Gurgaon, India', Timestamp('2024-03-02 19:00:00')]
2024-03-02 14:14:39,943 - __main__ - INFO - The hourly data has been written to data/cron_job_data/vikas_sadan_cron_output.csv with Timestamp: 2024-03-02 19:00:00
2024-03-02 15:15:41,399 - __main__ - INFO - setData function - AQI is not an integer. Exception ValueError has occured for station=> nise gwal
2024-03-02 15:15:42,074 - __main__ - INFO - station=> Sector-51, Gurugram, result => ['85', 'Sector-51, Gurugram, India', Timestamp('2024-03-02 20:00:00')]
2024-03-02 15:15:42,074 - __main__ - INFO - The hourly data has been written to data/cron_job_data/sector_51_cron_output.csv with Timestamp: 2024-03-02 20:00:00
2024-03-02 15:15:42,754 - __main__ - INFO - station=> Sector-51, Gurugram, result => ['85', 'Sector-51, Gurugram, India', Timestamp('2024-03-02 20:00:00')]
2024-03-02 15:15:42,754 - __main__ - INFO - The hourly data has been written to static/data/sector_51_cron_output.csv with Timestamp: 2024-03-02 20:00:00
2024-03-02 15:15:43,432 - __main__ - INFO - station=> Teri Gram, result => ['158', 'Teri Gram, Gurugram, India', Timestamp('2024-03-02 20:00:00')]
2024-03-02 15:15:43,432 - __main__ - INFO - The hourly data has been written to data/cron_job_data/teri_gram_cron_output.csv with Timestamp: 2024-03-02 20:00:00
2024-03-02 15:15:44,103 - __main__ - INFO - station=> Vikas Sadan, result => ['158', 'Vikas Sadan Gurgaon, Gurgaon, India', Timestamp('2024-03-02 20:00:00')]
2024-03-02 15:15:44,103 - __main__ - INFO - The hourly data has been written to data/cron_job_data/vikas_sadan_cron_output.csv with Timestamp: 2024-03-02 20:00:00
2024-03-02 16:19:53,185 - __main__ - INFO - setData function - AQI is not an integer. Exception ValueError has occured for station=> nise gwal
2024-03-02 16:19:53,860 - __main__ - INFO - station=> Sector-51, Gurugram, result => ['84', 'Sector-51, Gurugram, India', Timestamp('2024-03-02 21:00:00')]
2024-03-02 16:19:53,860 - __main__ - INFO - The hourly data has been written to data/cron_job_data/sector_51_cron_output.csv with Timestamp: 2024-03-02 21:00:00
2024-03-02 16:19:54,542 - __main__ - INFO - station=> Sector-51, Gurugram, result => ['84', 'Sector-51, Gurugram, India', Timestamp('2024-03-02 21:00:00')]
2024-03-02 16:19:54,542 - __main__ - INFO - The hourly data has been written to static/data/sector_51_cron_output.csv with Timestamp: 2024-03-02 21:00:00
2024-03-02 16:19:55,220 - __main__ - INFO - station=> Teri Gram, result => ['152', 'Teri Gram, Gurugram, India', Timestamp('2024-03-02 21:00:00')]
2024-03-02 16:19:55,220 - __main__ - INFO - The hourly data has been written to data/cron_job_data/teri_gram_cron_output.csv with Timestamp: 2024-03-02 21:00:00
2024-03-02 16:19:55,895 - __main__ - INFO - station=> Vikas Sadan, result => ['122', 'Vikas Sadan Gurgaon, Gurgaon, India', Timestamp('2024-03-02 21:00:00')]
2024-03-02 16:19:55,895 - __main__ - INFO - The hourly data has been written to data/cron_job_data/vikas_sadan_cron_output.csv with Timestamp: 2024-03-02 21:00:00
2024-03-02 17:13:50,975 - __main__ - INFO - setData function - AQI is not an integer. Exception ValueError has occured for station=> nise gwal
2024-03-02 17:13:51,615 - __main__ - INFO - station=> Sector-51, Gurugram, result => ['99', 'Sector-51, Gurugram, India', Timestamp('2024-03-02 22:00:00')]
2024-03-02 17:13:51,615 - __main__ - INFO - The hourly data has been written to data/cron_job_data/sector_51_cron_output.csv with Timestamp: 2024-03-02 22:00:00
2024-03-02 17:13:52,252 - __main__ - INFO - station=> Sector-51, Gurugram, result => ['99', 'Sector-51, Gurugram, India', Timestamp('2024-03-02 22:00:00')]
2024-03-02 17:13:52,252 - __main__ - INFO - The hourly data has been written to static/data/sector_51_cron_output.csv with Timestamp: 2024-03-02 22:00:00
2024-03-02 17:13:52,906 - __main__ - INFO - station=> Teri Gram, result => ['155', 'Teri Gram, Gurugram, India', Timestamp('2024-03-02 22:00:00')]
2024-03-02 17:13:52,906 - __main__ - INFO - The hourly data has been written to data/cron_job_data/teri_gram_cron_output.csv with Timestamp: 2024-03-02 22:00:00
2024-03-02 17:13:53,555 - __main__ - INFO - station=> Vikas Sadan, result => ['145', 'Vikas Sadan Gurgaon, Gurgaon, India', Timestamp('2024-03-02 22:00:00')]
2024-03-02 17:13:53,555 - __main__ - INFO - The hourly data has been written to data/cron_job_data/vikas_sadan_cron_output.csv with Timestamp: 2024-03-02 22:00:00
2024-03-02 18:21:52,274 - __main__ - INFO - setData function - AQI is not an integer. Exception ValueError has occured for station=> nise gwal
2024-03-02 18:21:52,736 - __main__ - INFO - station=> Sector-51, Gurugram, result => ['111', 'Sector-51, Gurugram, India', Timestamp('2024-03-02 23:00:00')]
2024-03-02 18:21:52,736 - __main__ - INFO - The hourly data has been written to data/cron_job_data/sector_51_cron_output.csv with Timestamp: 2024-03-02 23:00:00
2024-03-02 18:21:53,196 - __main__ - INFO - station=> Sector-51, Gurugram, result => ['111', 'Sector-51, Gurugram, India', Timestamp('2024-03-02 23:00:00')]
2024-03-02 18:21:53,196 - __main__ - INFO - The hourly data has been written to static/data/sector_51_cron_output.csv with Timestamp: 2024-03-02 23:00:00
2024-03-02 18:21:53,663 - __main__ - INFO - station=> Teri Gram, result => ['114', 'Teri Gram, Gurugram, India', Timestamp('2024-03-02 23:00:00')]
2024-03-02 18:21:53,664 - __main__ - INFO - The hourly data has been written to data/cron_job_data/teri_gram_cron_output.csv with Timestamp: 2024-03-02 23:00:00
2024-03-02 18:21:54,124 - __main__ - INFO - station=> Vikas Sadan, result => ['136', 'Vikas Sadan Gurgaon, Gurgaon, India', Timestamp('2024-03-02 23:00:00')]
2024-03-02 18:21:54,125 - __main__ - INFO - The hourly data has been written to data/cron_job_data/vikas_sadan_cron_output.csv with Timestamp: 2024-03-02 23:00:00
2024-03-02 19:12:35,330 - __main__ - INFO - setData function - AQI is not an integer. Exception ValueError has occured for station=> nise gwal
2024-03-02 19:12:36,007 - __main__ - INFO - station=> Sector-51, Gurugram, result => ['100', 'Sector-51, Gurugram, India', Timestamp('2024-03-03 00:00:00')]
2024-03-02 19:12:36,007 - __main__ - INFO - The hourly data has been written to data/cron_job_data/sector_51_cron_output.csv with Timestamp: 2024-03-03 00:00:00
2024-03-02 19:12:36,681 - __main__ - INFO - station=> Sector-51, Gurugram, result => ['100', 'Sector-51, Gurugram, India', Timestamp('2024-03-03 00:00:00')]
2024-03-02 19:12:36,681 - __main__ - INFO - The hourly data has been written to static/data/sector_51_cron_output.csv with Timestamp: 2024-03-03 00:00:00
2024-03-02 19:12:37,359 - __main__ - INFO - station=> Teri Gram, result => ['100', 'Teri Gram, Gurugram, India', Timestamp('2024-03-03 00:00:00')]
2024-03-02 19:12:37,359 - __main__ - INFO - The hourly data has been written to data/cron_job_data/teri_gram_cron_output.csv with Timestamp: 2024-03-03 00:00:00
2024-03-02 19:12:38,019 - __main__ - INFO - station=> Vikas Sadan, result => ['132', 'Vikas Sadan Gurgaon, Gurgaon, India', Timestamp('2024-03-03 00:00:00')]
2024-03-02 19:12:38,019 - __main__ - INFO - The hourly data has been written to data/cron_job_data/vikas_sadan_cron_output.csv with Timestamp: 2024-03-03 00:00:00
